openfl-gramine/openfl.manifest.template

```diff
@@ -38,6 +38,10 @@ fs.mount.workspace.type = "chroot"
 fs.mount.workspace.path = "/workspace"
 fs.mount.workspace.uri = "file:/workspace"
 
+fs.mount.tmp.type = "tmpfs"
+fs.mount.tmp.path = "/tmp"
+fs.mount.tmp.uri = "file:/tmp"
+
```

Comment:
These changes are already in develop

Reply:
Thanks, I've rebased the manifest to the latest develop commit